.github/workflows/workflow.yaml

```diff
@@ -116,11 +116,11 @@ jobs:
         run: flutter --version | grep '3.3.10'
         shell: bash
 
-  test_print_output:
-    runs-on: macos-13
+  test_print_output_x64:
+    runs-on: ubuntu-latest
```

Comment:
Does the `runs-on` affect if the tests will be successful or not?

Reply:
Currently, flutter for Linux is only available for x64. 

https://docs.flutter.dev/release/archive

Also, the GitHub Actions ubuntu is currently only available for x64. However, in the future it will be possible to run it for arm64 as well.

https://github.blog/news-insights/product-news/arm64-on-github-actions-powering-faster-more-efficient-build-systems/

Therefore, if ubuntu-latest switches to arm64, this test will fail. (Probably, we cannot get the old sdk that does not support arm64.) This setting would be as expected, since it can detect that ubuntu-latest is switching to arm64 and can reveal what features flutter-action can support.